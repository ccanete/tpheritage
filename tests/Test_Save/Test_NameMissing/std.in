SAVE 
SAVE test 1
EXIT