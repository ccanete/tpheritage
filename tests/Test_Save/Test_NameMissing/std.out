ERR
# Invalide file name.
ERR
# Invalide file name.

# The program is being closed.